app/src/main/res/layout/activity_location_onboarding.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/onboarding_content_root"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:gravity="center_horizontal|fill_vertical"
+  android:orientation="vertical">
+
+  <Button
+    android:id="@+id/skip_button"
+    style="@style/Onboarding.Skip"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="end"
+    android:layout_marginTop="@dimen/onboarding_skip_button_margin_vertical"
+    android:layout_marginEnd="@dimen/onboarding_skip_button_margin_end"
+    android:layout_marginBottom="@dimen/onboarding_skip_button_margin_vertical"
+    android:text="@string/onboarding_skip_button" />
+
+  <ImageView
+    style="@style/Onboarding.Logo"
+    android:layout_width="wrap_content"
+    android:layout_height="@dimen/onboarding_logo_height"
+    android:layout_marginStart="@dimen/onboarding_content_margin_horizontal"
+    android:layout_marginTop="@dimen/onboarding_logo_margin_top"
+    android:layout_marginEnd="@dimen/onboarding_content_margin_horizontal"
+    android:contentDescription="@null"
+    android:src="@drawable/google_primary" />
```

Comment:
since we already had issues with vector drawables can we just use `app:srcCompat` ?